src/content/protocols/kodiak-v3.md

```diff
@@ -0,0 +1,230 @@
+
+## Exit Window
+
+The protocol has no exit window mechanisms, allowing immediate execution of all critical functions by multi-sig wallets.
+
+> Exit Window score: High
+>
+> Justification: The protocol has no exit window mechanisms, and all critical functions can be executed immediately by multi-sig wallets. Combined with the High upgradeability risk, this represents a significant risk to user funds as there is no protection period for users to exit before potentially malicious changes take effect.
+
+
+## Accessibility
+Kodiak is accessible through only a frontend [kodiak.finance](https://app.kodiak.finance/), no code is available on Github.
+
+> Accessibility score: High
+
+# Technical Analysis
+
+## Contracts
+
+| Contract Name       | Address                                                                                                               |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| UniswapV3Factory    | [0xD84CBf0B02636E7f53dB9E5e45A616E05d710990](https://berascan.com/address/0xD84CBf0B02636E7f53dB9E5e45A616E05d710990) |
+| SwapRouter          | [0xEd158C4b336A6FCb5B193A5570e3a571f6cbe690](https://berascan.com/address/0xEd158C4b336A6FCb5B193A5570e3a571f6cbe690) |
+| SwapRouter02        | [0xe301E48F77963D3F7DbD2a4796962Bd7f3867Fb4](https://berascan.com/address/0xe301E48F77963D3F7DbD2a4796962Bd7f3867Fb4) |
+| QuoterV2            | [0x644C8D6E501f7C994B74F5ceA96abe65d0BA662B](https://berascan.com/address/0x644C8D6E501f7C994B74F5ceA96abe65d0BA662B) |
+| MixedRouteQuoterV1  | [0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB](https://berascan.com/address/0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB) |
+| TickLens            | [0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48](https://berascan.com/address/0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48) |
+| KodiakIslandFactory | [0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6](https://berascan.com/address/0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6) |
+| IslandRouter        | [0x679a7C63FC83b6A4D9C1F931891d705483d4791F](https://berascan.com/address/0x679a7C63FC83b6A4D9C1F931891d705483d4791F) |
+| KodiakIsland        | [0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590](https://berascan.com/address/0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590) |
+| FarmFactory         | [0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF](https://berascan.com/address/0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF) |
+| KodiakFarm          | [0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa](https://berascan.com/address/0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa) |
+| PandaFactory        | [0xac335fe675699b0ce4c927bdaa572eb647ed9f02](https://berascan.com/address/0xac335fe675699b0ce4c927bdaa572eb647ed9f02) |
```

Comment:
missing contracts

- rewardvault e.g https://berascan.com/address/0x45325df4a6a6ebd268f4693474aaaa1f3f0ce8ca#code 
- factory of rewardvaults (0x94ad6ac84f6c6fba8b8ccbd71d9f4f101def52a8 - a ERC1967Proxy)
- PandaToken
- UniswapV3Pool